gpjax/parameters.py

```diff
@@ -278,12 +291,28 @@ def build_trainables(params: tp.Dict) -> tp.Dict:
     return prior_container
 
 
-def stop_grad(param: tp.Dict, trainable: tp.Dict):
+def build_trainables_false(params: Dict) -> Dict:
+    """Construct a dictionary of trainable statuses for each parameter. By default, every parameter within the model is NOT trainable.
+
+    Args:
+        params (Dict): The parameter set for which trainable statuses should be derived from.
+
+    Returns:
+        Dict: A dictionary of boolean trainability statuses. The dictionary is equal in structure to the input params dictionary.
+    """
+    # Copy dictionary structure
+    prior_container = deepcopy(params)
+    # Set all values to zero
+    prior_container = jax.tree_util.tree_map(lambda _: False, prior_container)
+    return prior_container
```

Comment:
I still don't like that this is two functions - it it feels like an argument would resolve it:
```python
def build_trainables_true(params: Dict, status: bool = False) -> Dict:
    """Construct a dictionary of trainable statuses for each parameter. By default, every parameter within the model is trainable.
    Args:
        params (Dict): The parameter set for which trainable statuses should be derived from.
        status (bool): Whether or not gradients should be taken with respect to this parameter.
    Returns:
        Dict: A dictionary of boolean trainability statuses. The dictionary is equal in structure to the input params dictionary.
    """
    # Copy dictionary structure
    prior_container = deepcopy(params)
    # Set all values to zero
    prior_container = jax.tree_util.tree_map(lambda _: status, prior_container)
    return prior_container

Reply:
Resolved in cb45fbf.